composer.json

```diff
@@ -8,6 +8,7 @@
     ],
     "require": {
         "php": ">=8.1",
+        "codedruids/php-coding-standards": "^0.1.1",
```

Comment:
Hi, what is this dependency for? We try to narrow it down to as minimum dependency as possible :)

Reply:
It's to provide the custom Laravel fixer, which doesn't have its own official package by itself for the Fixer - I wish it did, I would have included that instead!

Reply:
Alternatively I could copy-paste the Fixer itself into ECS? It seemed like you wanted to keep the repo clear of specific fixers/sniffs though and only reference via packages.

Reply:
Just for the fixer that this dependency supplies, or for the setlist as well?

Reply:
@TomasVotruba which way would you like me to go with this?

Reply:
Remove the package please, so we don't depend on anything new.

Reply:
OK, done. 👍